### ISSUE TITLE
the difference of color_pc_node and color_pc_sync_node

### ISSUE THREAD

**Issue by QipiaoZhou** [GCaptainNemo/fusion-lidar-camera-ROS#4]
i think the two nodes use two difference methods to realize fusion(time synchronize).is that right?

**Comment by GCaptainNemo** [comment]
> i think the two nodes use two difference methods to realize fusion(time synchronize).is that right?

Yes, I want to test the time delay of different synchronization methods.

**Comment by QipiaoZhou** [comment]
> > i think the two nodes use two difference methods to realize fusion(time synchronize).is that right?
> 
> Yes, I want to test the time delay of different synchronization methods.

So which one has less delay?

**Comment by GCaptainNemo** [comment]
> > > i think the two nodes use two difference methods to realize fusion(time synchronize).is that right?
> > 
> > 
> > Yes, I want to test the time delay of different synchronization methods.
> 
> So which one has less delay?

I'm sorry. I forgot the conclusion. you can test on your platform.
